scripts/xspdb/README.en.md

```diff
@@ -0,0 +1,194 @@
+
+
+<div align="center">
+<img src="/.github/screenshot.png" alt="screenshot" width="800" />
+<br>
+Screenshot of the XSPdb
+</div>
+
+### Installation & Dependencies
+
+Clone repository and install dependencies:
+```bash
+git clone https://github.com/OpenXiangShan/XSPdb.git # Clone repository
+pip install -r XSPdb/requirements.txt                # Install dependencies
+pip install .                                        # Optional installation
+```
+
+If source integration is not required, use alternative pip installation:
+```bash
+pip3 install XSPdb@git+https://github.com/OpenXiangShan/XSPdb@master
+```
+
+### Quick Start
+
+After cloning, run test suite:
+```bash
+cd XSPdb
+make test
+```
+
+This command executes:
+- Downloads Xiangshan Python binaries
+- Fetches test binaries
+- Launches interactive debug via example/test.py
+
+Example output and interaction:
+```bash
+LD_PRELOAD=XSPython/xspcomm/libxspcomm.so.0.0.1 PYTHONPATH=. python3 example/test.py
+Using simulated 32768B flash
+[Info] reset complete
+> XSPdb/example/test.py(13)test_sim_top()
+-> while True:
+(XiangShan) # Enter interactive mode (use Tab for command list)
+(XiangShan)xui                               # Enter TUI mode
+(XiangShan)xload ready-to-run/microbench.bin # Load binary (Tab-compatible)
+(XiangShan)xistep                            # Step to next instruction commit
+(XiangShan)xstep 10000                       # Execute 10000 cycles
+```
+
+**Note**: XSPdb prioritizes `spike-dasm` for disassembly, falling back to `capstone`(limited instruction support).
+
+### Manual Testing
+
+Prerequisite: Build Xiangshan Python simulator ([build guide]()).
+
+Alternative: Download prebuilt binaries:
+```bash
+cd XSPdb
+wget https://github.com/OpenXiangShan/XSPdb/releases/download/v0.1.0-test/XSPython.tar.gz
+wget https://github.com/OpenXiangShan/XSPdb/releases/download/v0.1.0-test/ready-to-run.tar.gz
+tar xf XSPython.tar.gz
+tar xf ready-to-run.tar.gz
+```
+
+Launch test:
+```bash
+LD_PRELOAD=XSPython/xspcomm/libxspcomm.so.0.0.1 PYTHONPATH=. python3 example/test.py
+```
+
+**Node**: The reason for using `LD_PRELOAD` to load `xspcomm` in advance is to prevent version conflicts between the `system xspcomm` library and `the local xspcomm `packaged in XSPython.
+
+### Command-Line Mode (Non-Interactive):
+
+In addition to the interactive mode, XSPdb also provides a batch mode, allowing you to use it as a full-system simulator. The entry point is `emu.py`, for example:
+
+```bash
+./emu.py -i ready-to-run/microbench.bin
```

Comment:
thw PR should not include the `emu.py` flow

only `make pdb` / `make pdb-run` shoud be descripted in doc